gips/data/gpm/gpm.py

```diff
@@ -209,51 +232,63 @@ def fetch(cls, asset, tile, date):
             return [stage_fp]
         return []
 
+
 class gpmData(Data):
     """ A tile of data (all assets and products) """
     name = 'GPM'
     version = '1.0.0'
     Asset = gpmAsset
 
-    _projection = 'GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.2572326660159,' \
-                  'AUTHORITY["EPSG","7030"]], AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0],' \
+    _projection = 'GEOGCS["WGS 84",DATUM["WGS_1984",' \
+                  'SPHEROID["WGS 84",6378137,298.2572326660159,' \
+                  'AUTHORITY["EPSG","7030"]], AUTHORITY["EPSG","6326"]],' \
+                  'PRIMEM["Greenwich",0],' \
```

Comment:
This is fine to leave as it is, but here's another place you can avoid backslashes:

```
_projection = ('GEOGCS["WGS 84",DATUM["WGS_1984",'
               'SPHEROID["WGS 84",6378137,298.2572326660159,'
               'AUTHORITY["EPSG","7030"]], AUTHORITY["EPSG","6326"]],'
               'PRIMEM["Greenwich",0],')
```

That way two parens can govern the staement no matter how long it is instead of one backslash per line.